packages/jellyfish-api-core/__tests__/category/account/getAccountHistory.test.ts

```diff
@@ -44,8 +44,6 @@ describe('Account', () => {
     expect(history.txn).toStrictEqual(referenceHistory.txn)
     expect(history.txid).toStrictEqual(referenceHistory.txid)
     expect(history.type).toStrictEqual(referenceHistory.type)
-    expect(history.blockHash).toStrictEqual(referenceHistory.blockHash)
```

Comment:
Might be better to keep another history which in a minted block and non undefined block hash